### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.
- Present the user with a list of options to perform.
- Lets users choose an option.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    puts 'Enter the name of student:'
+    name = gets.chomp
+    puts 'Enter the age of the student'
+    age = gets.chomp
+    puts 'parent permission? [Y/N]:'
+    parent_permission = gets.chomp.downcase
+    student = Student.new(@class, age, name, parent_permission)
+    @people << student
+    puts 'Student created successfully'
+  end
```
Comment (Emmyn5600) [inline main.rb]: - [ ] In the classroom.rb file, you are importing the wrong file which is supposed to be `./student.rb` not `./Student.rb` which makes your application raise an error.

rb

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    puts 'Enter the name of the teacher:'
+    name = gets.chomp
+    puts 'Enter the age of the teacher'
+    age = gets.chomp
+    puts 'Enter the Specialization:'
+    specialization = gets.chomp
+    teacher = Teacher.new(specialization, age, name)
+    @people << teacher
+    puts 'Teacher created successfully'
+  end
```
Comment (Emmyn5600) [inline main.rb]: - [ ] Kindly, validate the input for the number to be a number, not text and vise-versa

- Age is a number, not a text
- Name is a text, not a number.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @Danieldotcomcoder 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

 ## Required Changes ♻️

- Check the comments under the review.

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by julie-ify — APPROVED** [review]
## Hi @Danieldotcomcoder, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**